apps/network-alerts/js/index.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
+// See reference TS in following document for GSM and CDMA:
+// [1] GSM: http://www.etsi.org/deliver/etsi_ts/123000_123099/123041/
+//     11.06.00_60/ts_123041v110600p.pdf, chapter 9.4.1.2.2 Message Identifier
```

Comment:
this section is specific to the messageId, so maybe it's better to keep the full PDF link here, but the chapter reference before `GSM_PRESIDENTIAL_ALERTS` ?

Also you can have the full URL without a jshint issue if you you don't have any other word before the URL.